chia/full_node/hint_store.py

```diff
@@ -0,0 +1,39 @@
+from typing import List, Tuple
+import aiosqlite
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.db_wrapper import DBWrapper
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class HintStore:
+    coin_record_db: aiosqlite.Connection
+    db_wrapper: DBWrapper
+
+    @classmethod
+    async def create(cls, db_wrapper: DBWrapper):
+        self = cls()
+        self.db_wrapper = db_wrapper
+        self.coin_record_db = db_wrapper.db
+        # the coin_id is unique, same hint can be used for multiple coins
+        await self.coin_record_db.execute(("CREATE TABLE IF NOT EXISTS hints(coin_id blob PRIMARY KEY,  hint blob)"))
```

Comment:
if we ever want to support multiple hints per coin, changing this schema may be the biggest hurdle

Reply:
good catch, I'll add another auto-increment id column there so that coin id is not unique

Reply:
added